clever_tools/setup.py

```diff
@@ -0,0 +1,9 @@
+from distutils.core import setup
+from catkin_pkg.python_setup import generate_distutils_setup
+ 
+setup_args = generate_distutils_setup(
+    packages=['flight_routines'],
```

Comment:
Shouldn't `flight_routines` be submodules from `clever_tools`? Why they should be seen as a global package?